.travis.yml

```diff
@@ -127,6 +130,20 @@ install:
         travis_retry sudo apt-get -qq -y install libegl1-mesa;
       fi;
 
+    # Install IPython
+    # No IPython for 2.6
+    # Install using conda with old version for 2.7
+    # Install using pip with new version for > 2.7
```

Comment:
we should use conda with new version for > 2.7, and system version `apt-get install python-ipython` or whatever for 2.7